From 1735a8484ef9ae336f8d607b56bda64c8af10c79 Mon Sep 17 00:00:00 2001
From: Trevor Woerner <trevor.woerner@linaro.org>
Date: Fri, 7 Feb 2014 04:07:17 +0100
Subject: [PATCH] qtwebkit: fix QA issue (bad RPATH)

Building qtwebkit causes a QA issue such that QtWebPluginProcess and
QtWebProcess contain bad RPATHs which point into the build location. This fix
adds a patch to not include the rpath.prf which causes this problem.

Signed-off-by: Trevor Woerner <trevor.woerner@linaro.org>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

Downloaded from:
https://github.com/meta-qt5/meta-qt5/blob/krogoth/recipes-qt/qt5/qtwebkit/
0001-qtwebkit-fix-QA-issue-bad-RPATH.patch

Signed-off-by: Gary Bisson <gary.bisson@boundarydevices.com>
---
 Tools/qmake/mkspecs/features/unix/default_post.prf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Tools/qmake/mkspecs/features/unix/default_post.prf b/Tools/qmake/mkspecs/features/unix/default_post.prf
index fd66af6..ef754c3 100644
--- a/Tools/qmake/mkspecs/features/unix/default_post.prf
+++ b/Tools/qmake/mkspecs/features/unix/default_post.prf
@@ -61,7 +61,6 @@ linux-*g++* {
     }
 }
 
-contains(TEMPLATE, app): CONFIG += rpath
 
 CONFIG(debug, debug|release)|force_debug_info {
     # Make ld not cache the symbol tables of input files in memory to avoid memory exhaustion during the linking phase.
-- 
2.7.0

